jsconfig.json

```diff
@@ -5,12 +5,17 @@
 		"target": "ES6",
 		"baseUrl": ".",
 		"paths": {
-			"*": [ "test/*", "server/*", "client/*" ]
+			"*": [
+				"test/*",
+				"server/*",
+				"client/*"
+			]
 		}
 	},
 	"exclude": [
 		"node_modules",
+		"**/node_modules/*",
 		"public",
 		"build"
 	]
-}
```

Comment:
can you keep the newline

Reply:
Prettier says no; do we need to update something in our Prettier fork?

Reply:
i didn't think we were prettifying `.json` files :/.
It doesn't really matter one way or the other. feel free to ship without keeping newline

Reply:
also, sidenote:  you can always skip auto-prettification by doing:
```bash
git commit -n 
```